app/views/users/_form.html.erb

```diff
@@ -11,8 +11,8 @@
     <%= f.text_field :lastname, required: true %>
   </div>
   <div>
-    <%= f.label :pseudo %>
-    <%= f.text_field :pseudo, required: true %>
+    <%= f.label :username %>
+    <%= f.text_field :username, required: true %>
```

Comment:
That got me thinking... Why do we even have a view to create a user, since all users come from the SSO?
Note: create an issue to remove it?